frame/assistants/shadow-liketh/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "shadow-liketh"
+version = "0.5.3"
+authors = ["Darwinia Network <hello@darwinia.network>"]
+description = "Darwinia bridger"
+repository = "https://github.com/darwinia-network/bridger"
+license = "MIT"
+documentation = "https://rust-docs.darwinia.network/bridger"
+homepage = "https://github.com/darwinia-network/bridger"
+include = [
+  "Cargo.toml",
+  "**/*.rs",
+  "README.md",
+  "LICENSE"
+]
+keywords = ["substrate", "darwinia"]
+readme = 'README.md'
+edition = "2021"
+
+[dependencies]
+tracing     = "0.1"
+thiserror   = "1.0"
+include_dir = "0.7.2"
+
+serde      = { version = "1.0", features = ["derive"] }
+serde_json = { version = "1.0" }
+serde-hex  = "0.1"
+serde-aux  = "0.6"
+arrayvec   = "0.7"
+
+codec      = { package = "parity-scale-codec", version = "2.3.1", features = ["derive", "full"] }
+
+gql_client = "1"
+
+reqwest    = { version = "0.11", features = ["rustls-tls", "json"] }
+mmr_client = { package = "merkle-mountain-range", git = "https://github.com/darwinia-network/shadow.git", branch = "add-mmr-client" }
```

Comment:
`merkle-mountain-range` is a independent third party lib, we use the customized branch because we modified it?

Reply:
Nope, the `merkle-mountain-range` package is defined by shadow
https://github.com/darwinia-network/shadow/blob/e60661c78078fedfccf26a799248d8e43b26dd7d/mmr-client/Cargo.toml#L2
but this folder name is `mmr-client` so I changed package's name to `mmr_client`